segments-guide.md

```diff
@@ -0,0 +1,327 @@
+
+```
+
+{
+  "id": "s300000022_5bb7c94e80f0073611afb35c",
+  "name": "test create",
+  "description": "",
+  "rsid": "sistr2.beta",
+  "owner": {
+    "id": 596983
+  }
+}
+
+```
+
+## Deleting a Segment
+
+If you delete a segment, it is hidden from all users in all menus. Additionally, it is no longer returned when retrieving multiple segments in an API call. However, reports and other sources that reference the deleted segment are still able to use it. You can also continue to request it from the single segment API endpoint.
+
+### Example Request
+
+The following example deletes segment `s300000022_5bb7c94e80f0073611afb35c`:
+
+```
+curl -X DELETE "https://analytics.adobe.io/api/obueng0/segments/s300000022_5bb7c94e80f0073611afb35c?locale=en_US" -H "x-api-key: {OAUTHTOKEN}" -H "x-proxy-global-company-id: {COMPANYID}" -H "Authorization: Bearer {ACCESSTOKEN}" -H "Accept: application/json" -H "Content-Type: application/json"
+
+```
+
+### Example Response
+
+The following response shows the deletion status for segment `s300000022_5bb7c94e80f0073611afb35c`:
+
+```
+
+{
+	"status":"ok"
+}
+
+```
+
+## Updating a Segment
+
+You can edit existing segments with the `PUT /segments/{id} endpoint`. Some fields cannot be edited on a segment with the PUT endpoint, including `tags`, `compatibility`, and `reportSuiteName`. Other fields, including `owner`, `name`, `description`, `rsid`, and `definition`, can be edited.
+
+The `PUT` endpoint also supports partial updates. This means that instead of sending the entire JSON object to the API, you can simply send fields that you want to update. For example, if you only want to update the name, use the JSON `{"name":"Updated name"}`. The entire object is returned in the response, modified by the requested expansions.
+
+### Example Request
+
+The following example updates the segment `s300000022_5bb7c94e80f0073611afb35c` with a new name:
+
+```
+
+curl -X PUT "https://analytics.adobe.io/api/obueng0/segments/s300000022_5bb7c94e80f0073611afb35c?locale=en_US" -H "x-api-key: {OAUTHTOKEN}" -H "x-proxy-global-company-id: {COMPANYID}" -H "Authorization: Bearer {ACCESSTOKEN}" -H "Accept: application/json" -H "Content-Type: application/json" -d "{NAME}":"{NAME UPDATE}"
+
+```
+
+### Example Response
+
+The following example shows the response with updated name data:
+
+```
+
+{
+  "id": "s300000022_5bb7c94e80f0073611afb35c",
+  "{NAME}": "{NAME UPDATE}",
+  "description": "",
+  "rsid": "sistr2.beta",
+  "owner": {
+    "id": 596983
+  }
+}
+
+```
+
+## Validate Endpoint
+
+Because report suites can have different configurations, variables, or metrics, one segment that is valid in one report suite may not be valid in another. To determine which segments to use in different report suites, you can use the `validate` endpoint. This endpoint allows you to `POST` a definition along with a target `rsid`. The validate endpoint responds with compatibility information on the segment.
```

Comment:
Maybe this is the place to describe the relationship of rsids to segments.

Reply:
I put the note previously, but if, after reading the note, you think it should go here, just comment.